PacketGen.py

```diff
@@ -1,26 +1,18 @@
-import argparse
```

Comment:
I think we'd still like to have argparse and allow the user to pass a directory path. '.' on line 9 can be the default